expat/lib/xmlparse.c

```diff
@@ -63,6 +63,10 @@
 
 #include "expat_config.h"
 
+#if ! defined(XML_CONTEXT_BYTES) || (1 - XML_CONTEXT_BYTES - 1 == 2) || (XML_CONTEXT_BYTES + 0 < 0)
```

Comment:
Doesn't clang-format want to reformat this?

Reply:
No, I was just surprised that clang-format didn't break the `#error` line. But if it doesn't, we're all good.